nmoscommon/auth/constants.py

```diff
@@ -0,0 +1,10 @@
+# Constants for nmos-oauth defining file and directory locations
+import os
+
+NMOSOAUTH_DIR = '/var/nmosoauth'  # LINUX ONLY
```

Comment:
Are these really configuration options? If so should they go in nmoscommonconfig?